jobs/Frontend/movies-work/src/config/constants.tsx

```diff
@@ -0,0 +1,7 @@
+const Constants = {
+  API_URL: "https://api.themoviedb.org",
+  API_KEY: "03b8572954325680265531140190fd2a",
```

Comment:
Hey, the API KEY doesn't belong to here, or does it? ☺️  (fellow candidate here)